randovania/games/dread/json_data/Cataris.txt

```diff
@@ -124,7 +124,9 @@ Extra - asset_id: collision_camera_001
   * Layers: default
   * Slide Tunnel to Underlava Puzzle Room 2/Tunnel to Dropdown Pit
   > Tunnel to Above Z-57 Fight
-      Space Jump or Simple IBJ
+      Any of the following:
+          Simple IBJ or Use Spin Boost
+          Speed Booster and Disabled Door Lock Randomizer and Can Slide
```

Comment:
This one doesnt need it. You can just charge it in the hallway, which does not have doors. If anything it would be Disabled Block Randomizer if we actually do plan on doing that.